charts/plgd-hub/templates/_helpers.tpl

```diff
@@ -162,9 +162,9 @@ If release name contains chart name it will be used as a full name.
     {{- $raClientTls := .Values.coapgateway.clients.resourceAggregate.grpc.tls.certFile }}
     {{- $rdClientTls := .Values.coapgateway.clients.resourceDirectory.grpc.tls.certFile }}
     {{- if and $natsTls $authClientTls $raClientTls $rdClientTls }}
-    {{- printf "false" }}
+    {{- printf "" -}}
```

Comment:
why the "-" at the end "-}}", all other lines do not have it, does it mean something or it is optional?